syntax = "proto3";

message IAuthRequest {
    string login = 1;
    string password = 2;
}
